macos/FluentUI/Button/Button.swift

```diff
@@ -352,6 +358,14 @@ open class Button: NSButton {
 	}
 
 	private func setColorValues(forStyle: ButtonStyle, accentColor: NSColor) {
+		if increaseContrastEnabled {
+			// Use "UI Element Colors" System Colors which respond correctly to accessibility
+			// settings like increase contrast
```

Comment:
You could also link the Apple HIG and/or apple docs here. Or just say "Use system colors"